agent/consul/replication.go

```diff
@@ -59,7 +59,12 @@ func NewReplicator(config *ReplicatorConfig) (*Replicator, error) {
 		return nil, fmt.Errorf("Cannot create the Replicator without a Delegate set in the config")
 	}
 	if config.Logger == nil {
```

Comment:
Repeat of the github.com/hashicorp/consul/lib helper function idea from earlier.